docs/components/receive.md

```diff
@@ -66,6 +74,13 @@ The example content of `hashring.json`:
 With such configuration any receive listens for remote write on `<ip>10908/api/v1/receive` and will forward to correct one in hashring if needed
 for tenancy and replication.
 
+### Enabling Routing/Ingestion for Thanos Receiver
+
+We have not added yet another flag for controlling this behaviour. We have relied on the below convention to enable ingestion/routing -
+* **Ingestion and Distribution** - Make sure `--receive.local-endpoint` and `--receive.hashrings` / `--receive.hashrings-file` are provided.
+* **Ingestion** - Since routing behaviour is not needed, just provide `--receive.local-endpoint` only.
+* **Distribution** - For routing functionality, just provide one of the hashring flags - `--receive.hashrings` / `--receive.hashrings-file`. Don't provide local endpoint, as it would enable ingestion also.
```

Comment:
```suggestion
* **Distribution** - For routing functionality, provide one of the hashring configuration flags: either `--receive.hashrings` or `--receive.hashrings-file`. Do not specify a local endpoint, as doing so would enable ingestion.
```

Reply:
One last thing comes to mind: we sometimes call it "distribution" and other times call it "request routing," or only "routing." For clarity, we should stick to one term and use it consistently.

Reply:
Lets use distribution throughout, sounds good to me.